shared/js/background/https.es6.js

```diff
@@ -163,7 +163,7 @@ class HTTPS {
         }
 
         // Only deal with http calls
-        if (urlObj.protocol !== 'http:') {
+        if (urlObj.protocol !== 'http:' && urlObj.protocol !== 'ws:') {
```

Comment:
Update the comment above.

Perhaps the following is clearer:
```suggestion
        if (!["http:", "ws:"].includes(urlObj.protocol)) {
```